mods/cnc/maps/nod07c/rules.yaml

```diff
@@ -0,0 +1,139 @@
+World:
+	LuaScript:
+		Scripts: nod07c.lua
+	MusicPlaylist:
+		StartingMusic: justdoit
+		VictoryMusic: nod_win1
+	MissionData:
+		Briefing: GDI has left themselves open to the deadliest weapon of all, information.\n\nOur spies tell us the GDI base in your area contains an advanced attack helicopter, the Orca.\n\nSteal this weapon, then fly to the nearby village and destroy it.\n\nThe attack will be blamed on GDI.
+		BriefingVideo: nod7b.vqa
+		StartVideo: bcanyon.vqa
+		WinVideo: x.vqa
```

Comment:
x.vqa doesn't exist. x is the placeholder for 'no vqa' in td .inis, iirc, though. Our map import used to do that correctly...